src/components/views/elements/CopyableText.tsx

```diff
@@ -42,7 +43,12 @@ const CopyableText: React.FC<IProps> = ({ children, getTextToCopy }) => {
         }
     };
 
-    return <div className="mx_CopyableText">
+    let classNames = "mx_CopyableText";
+    if (border) {
+        classNames += " mx_CopyableText_border";
+    }
```

Comment:
can we use the classNames utility here please

Reply:
Yes, we can!